web/src/components/PoiFilters.tsx

```diff
@@ -60,8 +62,17 @@ const StyledButton = styled(TextButton)`
 `
 
 const StyledIcon = styled(Icon)`
-  width: 24px;
-  height: 24px;
+  width: 30px;
+  height: 30px;
```

Comment:
:wrench: Am I assuming correctly that you changed this to fix the issue that the width of the checkbox is smaller than actually set? I think it would be better to set `min-width` instead:
```suggestion
  min-width: 24px;
```

Reply:
The size of the checkbox was increased from 24px to 30px so that it appears larger as suggested in the issue. In the mobile view, the checkbox also looked better to me, so I just increased it. 
The problem with the checkbox could not be reproduced by you, so I think it might just be a problem on my device :)

Reply:
Thank you! I guess the instructions where not clear and I previously did not get the underlying issue why it looked the way it did. Thanks for your patience here :)